orbbec_acquire/util.py

```diff
@@ -315,34 +316,21 @@ def start_recording(
             ir_data = np.resize(ir_data, (height, width, 1))
 
             # info about ir for rendering
-            data_type = np.uint16
-            image_dtype = cv2.CV_16UC1
-            max_data = 65535
-            cv2.normalize(ir_data, ir_data, 0, max_data, cv2.NORM_MINMAX, dtype=image_dtype)
-            ir_data = ir_data.astype(data_type)
+            # data_type = np.uint16
+            # image_dtype = cv2.CV_16UC1
+            # max_data = 65535
+            # cv2.normalize(ir_data, ir_data, 0, max_data, cv2.NORM_MINMAX, dtype=image_dtype)
+            # ir_data = ir_data.astype(data_type)
```

Comment:
I prefer to keep it. See https://github.com/orbbec/pyorbbecsdk/blob/main/examples/infrared_viewer.py

Reply:
My understanding of `cv2.normalize` with the `cv2.NORM_MINMAX` flag is that it changes the IR values from 0-65535 to 0-1, which doesn't seem compatible with your display function. Also, you're normalizing before saving, which suggests that the saved movie might only contain 0s and 1s. When saving data for analysis, it is better to minimize data processing at this step.

Reply:
I might need some test to look into this.